### ISSUE TITLE
Timezone problemas

### ISSUE THREAD

**Issue by alexbannon** [ga-wdi-exercises/project2#181]
My migrations read t.datetime :created_at and when I do a new post, it is putting me at +5 in development and +4 in production on heroku.

I've tried the javascript fix as well as messing around in config to no avail. In talking with others, they are not having this issue just by putting in t.datetime and doing nothing else (and rails handles it automatically).

**Comment by amaseda** [comment]
Do you mean +4 and +5 relative to UTC?

Came across this Stack Overflow post with a lot of info. Can come take a look if this doesn't help. http://stackoverflow.com/questions/14252579/models-datetime-field-is-stored-differently-on-heroku-from-localhost

**Comment by alexbannon** [comment]
Yeah I found that pretty early in my struggles and it was helpful to understand how things are stored, but the weird part is, without me doing anything, it seems to auto adjust user input to UTC. From what I gather, it shouldn't be doing this, and the couple people I've compared with said they only did t.datetime :created_at and nothing else and it stores and displays the correct user time.

Anyway, totally not a dealbreaker. Just annoying. If there's no easy solution like change the migration then I'm gonna put it on the backburner anyway.
